lib/lotus/validations/error.rb

```diff
@@ -63,21 +60,16 @@ class Error
 
       # Initialize a validation error
       #
-      # @param attribute_name [Symbol] the name of the attribute
       # @param validation [Symbol] the name of the validation
       # @param expected [Object] the expected value
       # @param actual [Object] the actual value
-      # @param namespace [String] the optional namespace
       #
       # @since 0.1.0
       # @api private
-      def initialize(attribute_name, validation, expected, actual, namespace = nil)
-        @attribute_name = attribute_name.to_s
+      def initialize(validation, expected, actual)
```

Comment:
Ok, thanks for a great examples, the idea behind it is now much more clear for me! I'm gonna work now on getting back to this structure.